client/apps/edit/components/thumbnail/index.coffee

```diff
@@ -43,3 +43,22 @@ module.exports = class EditThumbnail extends Backbone.View
       $('.edit-use-article-title').hide()
     else
       $('.edit-use-article-title').show()
+
+  setupEmailMetadata: ->
+    new ImageUploadForm
+      el: $('#edit-email-upload')
+      src: @article.get('email_metadata')?.small_image_url
+      remove: =>
+        emailMetadata = @article.get('email_metadata') || {}
+        emailMetadata.large_image_url = ''
+        emailMetadata.small_image_url = ''
+        $('.edit-email-large-image-url span').text ''
+        $('.edit-email-small-image-url span').text ''
+        @article.save email_metadata: emailMetadata
+      done: (src) =>
+        emailMetadata = @article.get('email_metadata') || {}
+        emailMetadata.large_image_url = crop(src, { width: 1280, height: 960 } )
```

Comment:
Currently Positron just stores original urls and lets the client-app use Embedly/Gemini for resizing. I think it would be better to consolidate this to just `image_url` rather than save embedly urls to the article. That is unless I'm missing some key piece of context like the "client app" in this case is Mailchimp which can't easily integrate embedly—in which case I still think it would be good to store the original, pre-cropped image under `image_url` along-side this.